### PR TITLE
Update about-your-personal-dashboard.md

### PR DESCRIPTION
**Pull request by mrfresh63**
### Why:

Closes:

### Check off the following:

- [ ] A subject matter expert (SME) has reviewed the technical accuracy of the content in this PR. In most cases, the author can be the SME. Open source contributions may require a SME review from GitHub staff.
- [ ] The changes in this PR meet [the docs fundamentals that are required for all content](http://docs.github.com/en/contributing/writing-for-github-docs/about-githubs-documentation-fundamentals).
- [ ] All CI checks are passing.

### CODE REVIEW

**Review by T-droi — COMMENTED** [review]
Hii

**Comment by T-droi** [comment]
Unsubscribe

File: content/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-user-account-settings/about-your-personal-dashboard.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
----
+---https://www.facebook.com/share/1EdNY93Xky/?mibextid=LQQJ4d
```
Comment (Farnoosh85) [inline content/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-user-account-settings/about-your-personal-dashboard.md]: repository_dispatch

**Review by Farnoosh85 — COMMENTED** [review]
[Uploading Update Actions discussion category template · community_community@2438afc.txt…]()

**Comment by Farnoosh85** [comment]
For example, the following workflow triggers on the repository_dispatch event and uses information from the event payload to build the matrix. When a repository dispatch event is created with a payload like the one below, the matrix version variable will have a value of [12, 14, 16]. For more information about the repository_dispatch trigger
[Update Actions discussion category template · community_community@2438afc.txt](https://github.com/user-attachments/files/17979053/Update.Actions.discussion.category.template.community_community%402438afc.txt)

**Review by Farnoosh85 — COMMENTED** [review]
For example, the following workflow triggers on the repository_dispatch event and uses information from the event payload to build the matrix. When a repository dispatch event is created with a payload like the one below, the matrix version variable will have a value of [12, 14, 16]. For more information about the repository_dispatch trigger

**Pull request closed without merging**